### DISCUSSION TITLE
How to virtually press the Hue Bridge button (Link Button)

### DISCUSSION THREAD

**Discussion by BlackCatPeanut** [Foddy/node-red-contrib-huemagic#235] [Q&A] [answered]
Can anyone share an example flow of how to trigger a virtual press of the Hue Bridge Button (link button) as listed on the v3.0 features list?

**Comment by Foddy** [comment, marked as answer]
This is unfortunately no longer available. It was also not available with the v3 for long time, since Philips decided to restrict this feature in their updated bridge firmwares.

Anyways, thanks for the hint. I removed it from the readme.
